packages/compass-web/package.json

```diff
@@ -67,6 +67,7 @@
     "@mongodb-js/compass-aggregations": "^9.71.0",
     "@mongodb-js/compass-app-registry": "^9.4.18",
     "@mongodb-js/compass-app-stores": "^7.55.0",
+    "@mongodb-js/compass-assistant": "^1.0.0",
```

Comment:
by the way, how come Compass-Web only has a bunch of dev-dependencies?

Reply:
This code is packaged with webpack, so it's not a "production" dependency that needs to be installed with the package